### PR TITLE
Refactoring application_helper.rb

### PR DESCRIPTION
**Pull request by araslanov-e**
## What type of PR is this? (check all applicable)
- [x] Refactor
- [ ] Feature
- [ ] Bug Fix

## Description
Refactoring application_helper.rb
## Added to documentation?
  - [ ] docs.dev.to
  - [ ] readme
  - [x] no documentation needed

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -49,27 +51,20 @@ def icon(name, pixels = "20")
...
+
+    "https://res.cloudinary.com/practicaldev/image/upload/#{postfix}"
   end
 
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
-    if Rails.env.development? && (url.blank? || !url.include?("http"))
-      return url
-    end
+    return if Rails.env.development? && (url.blank? || url.exclude?("http"))
+
```
Comment (rhymes) [inline app/helpers/application_helper.rb]: Doesn't this change the behavior? It returned "url" before, not it returns nil

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -49,27 +51,20 @@ def icon(name, pixels = "20")
...
+    }.fetch(name, 'v1456342953/star-in-black-of-five-points-shape_sor40l.png')
+
+    "https://res.cloudinary.com/practicaldev/image/upload/#{postfix}"
   end
 
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
-    if Rails.env.development? && (url.blank? || !url.include?("http"))
-      return url
-    end
+    return if Rails.env.development? && (url.blank? || url.exclude?("http"))
```
Comment (ocnly) [inline app/helpers/application_helper.rb]: Should be `return **url** if Rails...`

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (araslanov-e) [reply app/helpers/application_helper.rb]: Oops

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (araslanov-e) [reply app/helpers/application_helper.rb]: Sorry

**Comment by araslanov-e** [comment]
@rhymes @ocnly fixed

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -12,26 +12,28 @@ def view_class
   end
 
   def core_pages?
```
Comment (ocnly) [inline app/helpers/application_helper.rb]: I think the method's name should be singular.
Refer to [this comment](https://github.com/thepracticaldev/dev.to/pull/332#issuecomment-412296031)

Comment (araslanov-e) [reply app/helpers/application_helper.rb]: I too, but did not change

Comment (ocnly) [reply app/helpers/application_helper.rb]: @rhymes, what do you have to say on this?

Comment (araslanov-e) [reply app/helpers/application_helper.rb]: @ocnly see too https://github.com/thepracticaldev/dev.to/pull/332

**Pull request merged**